cmd/goal/clerk.go

```diff
@@ -563,6 +564,22 @@ func lsigFromArgs(lsig *transactions.LogicSig) {
 	lsig.Args = getProgramArgs()
 }
 
+func getProto(versArg string) config.ConsensusParams {
+	cvers := protocol.ConsensusCurrentVersion
+	if protoVersion != "" {
+		cvers = protocol.ConsensusVersion(protoVersion)
+	}
+	proto, ok := config.Consensus[cvers]
+	if !ok {
+		fmt.Fprintf(os.Stderr, "Invalid consensus version. Possible versions:\n")
+		for xvers := range config.Consensus {
+			fmt.Fprintf(os.Stderr, "\t%s\n", xvers)
+		}
+		os.Exit(1)
```

Comment:
nit: most of the goal code is using `reportErrorf` to print and exit.

Reply:
yes, but it does one print and then exit, and I needed many prints and then an exit